client/web/src/enterprise/batches/batch-spec/TabBar.story.tsx

```diff
@@ -86,7 +99,7 @@ const LOCAL_TABS: TabsConfig[] = [
     { key: 'preview', isEnabled: true, handler: { type: 'link' } },
 ]
 
-add('for a locally-executed spec', () => (
+export const ForALocallyExecutedSpec: Story = () => (
```

Comment:
What do you think about using simpler variable names in places where we use `storyName`? Applicable to other story names.

```suggestion
export const LocallyExecutedSpec: Story = () => (
```